### PR TITLE
Optimise object_get_length to use entryCache

### PR DESCRIPTION
**Pull request by janisozaur**
Somewhat more questionable follow-up to #3591.

### CODE REVIEW

File: src/object.h [hunk 1]
```diff
@@ -115,7 +115,7 @@ int object_load_chunk(int groupIndex, rct_object_entry *entry, int* chunk_size);
 void object_unload_chunk(rct_object_entry *entry);
 int object_get_scenario_text(rct_object_entry *entry);
 void object_free_scenario_text();
-int object_get_length(rct_object_entry *entry);
+uintptr_t object_get_length(rct_object_entry **entryCache, size_t index);
```
Comment (IntelOrca) [inline src/object.h]: This is only used in `object_list`, you might as well move it to there and make it `static`.

Comment (janisozaur) [inline src/object.h]: the "questionable" part stems from this issue – would it ever be used anywhere else? If not, I can move it there, if yes, perhaps it's worth leaving it as-is.

Should I move it then or leave it be?

Comment (IntelOrca) [inline src/object.h]: My second suggestion was to leave it and make a new function for your particular purpose.

Comment (janisozaur) [inline src/object.h]: updated

**Pull request merged**